rpc-server/src/middlewares.rs

```diff
@@ -0,0 +1,114 @@
+use crate::metrics::{METHODS_CALLS_COUNTER, REQUESTS_COUNTER};
+use actix_web::dev::{forward_ready, Service, ServiceRequest, ServiceResponse, Transform};
+use futures::future::LocalBoxFuture;
+use futures::StreamExt;
+use jsonrpc_v2::{Params, RequestObject};
+use near_jsonrpc::RpcRequest;
+use serde_json::Value;
+use std::future::{ready, Ready};
+
+pub struct RequestsCounters;
```

Comment:
I'd like to see docstrings describing what it is and what is does to avoid deciphering the code.